common/test/acceptance/pages/lms/ccx_dashboard_page.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+CCX coach dashboard page
+"""
+from bok_choy.promise import EmptyPromise
+from .course_page import CoursePage
+
+
+class CoachDashboardPage(CoursePage):
+    """
+    CCX coach dashboard, where ccx coach can manage a course.
+    """
+    url_path = "ccx_coach"
+
+    def is_browser_on_page(self):
+        """
+        check if ccx dashboard is open.
+        """
+        return self.q(css='div.instructor-dashboard-wrapper-2').present
+
+    def is_browser_on_enrollment_page(self):
```

Comment:
Does the URL change if you're on the enrollment page? If so, that should be it's own page object.